core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -131,4 +140,36 @@
     text-align: center;
     text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
   }
+
+  simple-editor-body .question-type-wrapper {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
+    margin: 0 0 20px -30px;
+    overflow: hidden;
+    position: absolute;
+    width: 200px;
+    z-index: 20;
+  }
+  simple-editor-body .oppia-question-button {
+    background-color: #fff;
+    border-color: #fff;
+    border-width: 0;
+    color: #000;
+    height: 55px;
+    min-width: 202px;
+  }
+  simple-editor-body .oppia-question-button:hover,.oppia-question-button:focus {
+    background-color: #e4e4e4;
+    color: #000;
+  }
+  simple-editor-body .modal-wrapper {
+    width: 100%;
+    height: 100%;
+    background: rgba(0,0,0,0);
```

Comment:
For readability's sake, it would be better if it was set to `black` instead of what is present, compatibility-wise, it should be fine because this is quite a [basic](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords) attribute and it's [compatible](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Browser_compatibility) with all reasonably modern browsers.